pytket/binders/pauli.cpp

```diff
@@ -237,6 +237,184 @@ PYBIND11_MODULE(pauli, m) {
           "The list of Pauli terms")
       .def("__eq__", &PauliStabiliser::operator==)
       .def("__ne__", &PauliStabiliser::operator!=);
+
+  py::class_<QubitPauliTensor>(
```

Comment:
`QubitPauliTensor` should appear in the  `pytket.pauli` docs as a python class right? I don't see it in the built API documentation.

Reply:
ah my bad, I didn't install a develop version of pytket before building the docs locally. Please ignore.